org.adempiere.base/src/org/compiere/model/MOrder.java

```diff
@@ -1465,6 +1465,13 @@ protected boolean afterSave (boolean newRecord, boolean success)
 					line.setC_Currency_ID(getC_Currency_ID());
 				line.saveEx();
 			}
+			MOrderPaySchedule[] orderPaySchedules = MOrderPaySchedule.getOrderPaySchedule(getCtx(),this.getC_Order_ID(),0,get_TrxName());
+			for(MOrderPaySchedule orderPaySchedule : orderPaySchedules) {
+				if(is_ValueChanged(MOrder.COLUMNNAME_AD_Org_ID)) {
```

Comment:
For performance is better to check this if before the for and the getOrderPaySchedule.  Getting the array and traveling it is unnecessary if the org didn't change

Reply:
Thanks @VitorHenri - still one more line above, the code is still executing the MOrderPaySchedule.getOrderPaySchedule unnecessarily when the org has not changed.